addons/web/static/src/core/autocomplete/autocomplete.xml

```diff
@@ -69,6 +69,9 @@
                                             <t t-call="{{ source.optionTemplate }}" />
                                         </t>
                                         <t t-else="">
+                                            <img t-if='option.value' t-attf-src="/web/image/{{option.model}}/{{option.value}}/avatar_128"
+                                                class="rounded me-1 o_avatar"
+                                            />
```

Comment:
Should not be here but given in props in the `CalendarFilterPanel.getAutoCompleteProps`.
```js
sources: [
    {
        placeholder: _t("Loading..."),
        options: (request) => this.loadSource(section, request),
        optionTemplate: "web.CalendarFilterPanel.autocomplete.options", // <====
    },
],
```